runners/spark/src/main/java/org/apache/beam/runners/spark/SparkPipelineOptions.java

```diff
@@ -71,6 +71,10 @@
   String getCheckpointDir();
   void setCheckpointDir(String checkpointDir);
 
+  @Description("JARs to be submitted to the cluster, Usually a shaded JAR with all dependencies. ")
+  String getJars();
```

Comment:
Why not allow the user to use `List<String>` since PipelineOptionsFactory automatically supports parsing `--myOption=A,B,C` or `--myOption=A --myOption=B --myOption=C` as `List<String>` or `String[]`.

See this test examples:
https://github.com/apache/beam/blob/master/sdks/java/core/src/test/java/org/apache/beam/sdk/options/PipelineOptionsFactoryTest.java#L1194